src/test/java/kitchenpos/application/OrderServiceTest.java

```diff
@@ -28,145 +37,190 @@
+        menuGroup = new MenuGroup("치킨");
+        menu = new Menu(1L, "후라이드치킨", BigDecimal.valueOf(16000), menuGroup);
+        order1 = new Order(orderTable1, Collections.singletonList(new OrderLineItem(menu, 2L)));
+        order2 = new Order(orderTable2, Collections.singletonList(new OrderLineItem(menu, 2L)));
+    }
+
     @DisplayName("주문 등록")
     @Test
     void create() {
-        when(menuDao.countByIdIn(Collections.singletonList(1L))).thenReturn(1L);
-        when(orderTableDao.findById(1L)).thenReturn(Optional.of(TableGenerator.newInstance(1L, 1L, 4, false)));
-        when(orderDao.save(any(Order.class))).thenAnswer(invocation -> {
+        when(orderTableRepository.findById(1L)).thenReturn(Optional.of(orderTable1));
+        when(menuRepository.findAllById(any())).thenReturn(
+            Collections.singletonList(menu)
+        );
+        when(orderRepository.save(any(Order.class))).thenAnswer(invocation -> {
             Order order = invocation.getArgument(0);
-            return OrderGenerator.newInstance(1L, order.getOrderTableId(), order.getOrderStatus(), order.getOrderedTime());
-        });
-        when(orderLineItemDao.save(any(OrderLineItem.class))).thenAnswer(invocation -> {
-            OrderLineItem orderLineItem = invocation.getArgument(0);
-            return OrderGenerator.newOrderLineItem(1L, orderLineItem.getOrderId(), orderLineItem.getMenuId(), orderLineItem.getQuantity());
+            return new Order(
+                1L,
+                order.getOrderTable(),
+                order.getOrderStatus(),
+                order.getOrderLineItems()
+            );
         });
+        when(orderLineItemRepository.saveAll(any())).thenAnswer(
+            invocation -> invocation.getArgument(0)
+        );
 
-        OrderLineItem orderLineItem = OrderGenerator.newOrderLineItem(1L, 1);
-        Order order = OrderGenerator.newInstance(1L, Collections.singletonList(orderLineItem));
-        Order actual = orderService.create(order);
+        OrderLineItemRequest orderLineItemRequest = new OrderLineItemRequest(1L, 1L);
+        OrderRequest orderRequest = new OrderRequest(
+            1L,
+            Collections.singletonList(orderLineItemRequest)
+        );
+        OrderResponse actual = orderService.create(orderRequest);
 
-        verify(orderDao, times(1)).save(order);
-        verify(orderLineItemDao, times(1)).save(orderLineItem);
-        assertThat(actual).usingRecursiveComparison()
-            .ignoringFields("id", "orderLineItems.seq", "orderLineItems.orderId")
-            .isEqualTo(order);
+        verify(orderRepository, times(1)).save(any());
+        verify(orderLineItemRepository, times(1)).saveAll(any());
+        assertThat(actual.getOrderStatus()).isEqualTo(OrderStatus.COOKING.name());
         assertThat(actual.getId()).isNotNull()
             .isEqualTo(actual.getOrderLineItems().get(0).getOrderId());
         assertThat(actual.getOrderLineItems()).hasSize(1);
```

Comment:
```suggestion
        assertThat(actual.getOrderLineItems()).hasSize(orderRequest.getOrderLineItems().size());
```

전반적으로, 왜 이러한 수치를 가지기를 예상할 수 있을지에 대한 힌트가 있는 것이 좋다고 생각이 들었습니다.
테스트 코드를 보면서 아 이래서 1개 밖에 없겠구나~를 추측해야하는 느낌이 있었습니다.

Reply:
하드 코딩 해버렸네요 😅 수정하겠습니다!